orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestrator.java

```diff
@@ -1380,6 +1384,19 @@ private void executeRecipes(GatewayConfig gatewayConfig, Set<Node> allNodes, Exi
         }
     }
 
+    private boolean isPhaseSlsExists(SaltStateService saltStateService, GatewayConfig gatewayConfig, RecipeExecutionPhase phase, SaltConnector sc) {
```

Comment:
nit: `does` sound better instead of `is` an trailing `s` for exist not necessary :)